generate.go

```diff
@@ -1,4 +1,38 @@
-//go:generate go-bindata -pkg=data -ignore=.*\.go -o data.go . ./benkeen/d3pie/ ./jiahuang/d3-timeline/ ./krisk/fuse/ ./zeroclipboard/zeroclipboard/ ./seiyria/bootstrap-slider/
-//go:generate go fmt .
+// +build generate
 
-package data
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+	pathpkg "path"
+
+	"github.com/shurcooL/go/vfs/httpfs/filter"
+	"github.com/shurcooL/vfsgen"
+)
+
+func main() {
+	ignore := func(fi os.FileInfo, path string) bool {
+		switch {
+		case len(fi.Name()) >= 2 && fi.Name()[0] == '.': // Skip .files/.folders (but not ".").
+			return true
+		case pathpkg.Ext(fi.Name()) == ".go":
+			return true
+		case path == "/README.md":
+			return true
+		default:
+			return false
+		}
+	}
+	var data http.FileSystem = filter.NewIgnore(http.Dir("."), ignore)
+
+	err := vfsgen.Generate(data, vfsgen.Options{
+		Filename:     "data.go",
+		PackageName:  "data",
+		VariableName: "Assets",
+	})
+	if err != nil {
+		log.Fatalln(err)
+	}
+}
```

Comment:
One bothersome thought is how verbose this is. We've gone from a few `go:generate` lines to an entire script here. I think we should focus on making this minimal so that we're not regressing.

### `Filename`, `PackageName`, and `VariableName`

Let's make these the defaults that vfsgen provides us with, so we can omit these options entirely. This would become just:

```
if err := vfsgen.Generate(data, vfsgen.Options{}); err != nil {
    log.Fatalln(err)
}
```

### The ignore function...

...is rather verbose too! I wonder if we couldn't add a [glob-based](http://golang.org/pkg/path/filepath/#Glob) helper function to `filter` or some other package? e.g. `func Glob(globs ...string) func(fi os.FileInfo, path string) bool` Such that we could write just:

```
var data http.FileSystem = filter.NewIgnore(http.Dir("."), filter.Glob(
    "*.", // Not sure of proper glob to match hidden files/folders, would need to test
    "*.go",
    "/README.md",
))
```